fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java

```diff
@@ -77,9 +78,9 @@ public void analyze(Analyzer analyzer) throws UserException {
         super.analyze(analyzer);
 
         // check auth
-        if (!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
+        if (!Env.getCurrentEnv().getAccessManager().checkDbPriv(ConnectContext.get(), InternalCatalog.INTERNAL_CATALOG_NAME,Strings.isNullOrEmpty(dbName)?analyzer.getDefaultDb():dbName,PrivPredicate.LOAD)) {
```

Comment:
analyzer.getDefaultDb() may be null, the authentication logic should be placed after the following code
```
if (Strings.isNullOrEmpty(dbName)) {
            dbName = analyzer.getDefaultDb();
            if (Strings.isNullOrEmpty(dbName)) {
                ErrorReport.reportAnalysisException(ErrorCode.ERR_NO_DB_ERROR);
          }
           ```